src/ontology/uberon-edit.obo

```diff
@@ -224246,6 +224246,119 @@ intersection_of: part_of UBERON:0001987 ! placenta
 relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
 property_value: dcterms-date 2022-06-27T09:30:52Z xsd:dateTime
 
+[Term]
+id: UBERON:8480009
+name: perianal space
+def: "The virtual spaces circumscribed by the mucosa lining of the anal canal and the muscular structures of the sphincter complex." [http://www.endoscopy-colon-explorer.com/anatomy-of-the-anal-region/]
+is_a: UBERON:0000464 ! anatomical space
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+relationship: part_of UBERON:0001353 ! anal region
+property_value: dcterms-date 2022-07-05T10:24:30Z xsd:dateTime
+
+[Term]
+id: UBERON:8480010
+name: subcutaneous perianal space
+def: "A perianal space located below the dentate line. Its borders are the skin of the anal verge below and the internal sphincter above." [http://www.endoscopy-colon-explorer.com/anatomy-of-the-anal-region/]
+is_a: UBERON:8480009 ! perianal space
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-07-05T10:36:29Z xsd:dateTime
+
+[Term]
+id: UBERON:8480011
+name: submucous perianal space
+def: "A perianal space located above the dentate line. It is delimited by the internal sphincter and by the anoderm." [http://www.endoscopy-colon-explorer.com/anatomy-of-the-anal-region/]
+is_a: UBERON:8480009 ! perianal space
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-07-05T10:41:02Z xsd:dateTime
+
+[Term]
+id: UBERON:8480012
+name: intersphincteric perianal space
+def: "A perianal space located between the internal and external anal sphincter." [http://www.endoscopy-colon-explorer.com/anatomy-of-the-anal-region/]
+synonym: "Eisenhammer’s space" EXACT [http://www.endoscopy-colon-explorer.com/anatomy-of-the-anal-region/]
+is_a: UBERON:8480009 ! perianal space
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-07-05T10:43:20Z xsd:dateTime
+
+[Term]
+id: UBERON:8480013
+name: ischiorectal perianal space
+def: "A perianal space outside the anal canal that is delimited externally by the perianal skin and the gluteus, internally by the external sphincter and above by the levator ani muscle and internal obturator muscle. The two contralateral ischiorectal spaces communicate posteriorly around anorectal region via deep postanal space, located anteriorly to the ano-coccygeal ligament." [http://www.endoscopy-colon-explorer.com/anatomy-of-the-anal-region/]
+synonym: "ischio-rectal perianal space" EXACT [http://www.endoscopy-colon-explorer.com/anatomy-of-the-anal-region/]
+is_a: UBERON:8480009 ! perianal space
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-07-05T10:46:32Z xsd:dateTime
+
+[Term]
+id: UBERON:8480015
+name: superior pelvi-rectal perianal space
```

Comment:
@bvarner-ebi I added the synonyms.
Apart from that, as I said above:
seems to me that in this book the spaces are not exactly classified the same.
The ischioanal fossa or ischiorectal fossa is divided into the perianal space and the ischiorectal space.
Then authors describe that the different anorectal spaces are:
1- the perianal space,
2- the ischiorectal space (or ischioanal space),
3- the intersphincteric space,
4- supra levator space (or pelvic-rectal space),
5- the deep post-anal space.

Reply:
@bvarner-ebi I took all the classification of https://www.ncbi.nlm.nih.gov/books/NBK535387/. "superior pelvi-rectal perianal space" may be the same as the 'supra elevator space' but I did not add "superior pelvi-rectal perianal space" as synoym because I am not 100% sure

Reply:
That is fine, @paulacalipho. If you find a source that makes it clear, you would add the synonym along with a dbxref annotation to support the synonym. If you are still unsure, you can solicit more input from the community and make a decision or you could defer for now and return if and when appropriate.

Reply:
@bvarner-ebi I would prefer to defer for now and leave only the book source, thanks